cleaners/miro.xml

```diff
@@ -18,26 +18,48 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Miro
+    @url http://www.getmiro.com/
+    @os Windows, Linux
+    @cleanerversion v2.0.0
+    @cleanerdate 2019-03-14
+    @cleanerby ??? & https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows XP
+    @testeddate 2019-03-12 (... with an old version of the cleaner!)
+    @testedby https://github.com/bleachbit/bleachbit/issues/460#issuecomment-471987298
+    @note 
+
 -->
 <cleaner id="miro">
   <label>Miro</label>
   <description>Multimedia viewer</description>
+  <var name="cache-profile">
+    <value search="glob" os="windows">$AppData\Participatory Culture Foundation\Miro\Support</value>
+    <value search="glob" os="linux">~/.miro</value>
+  </var>
   <option id="cache">
     <label>Cache</label>
     <description>Delete the cache</description>
-    <action command="delete" search="walk.files" path="~/.miro/icon-cache/"/>
-    <action command="delete" search="walk.files" path="~/.miro/mozilla/Cache/"/>
-    <action command="delete" search="walk.files" path="$APPDATA\Participatory Culture Foundation\Miro\Support\icon-cache\"/>
+    <action command="delete" search="walk.files" path="$$cache-profile$$/icon-cache/"/>
+    <action command="delete" search="walk.files" path="$$cache-profile$$/mozilla/Cache/"/>
   </option>
   <option id="logs">
     <label>Logs</label>
     <description>Delete the logs</description>
     <action command="delete" search="file" path="~/.miro/miro-downloader-log"/>
     <action command="delete" search="file" path="~/.miro/miro-log"/>
     <action command="delete" search="file" path="~/.miro/miro-log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro-downloader.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro_Download_Daemon_a.txt"/>
+    <action command="delete" search="file" path="$Temp\Miro-downloader.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log.1"/>
+    <action command="delete" search="file" path="$Temp\Miro_Download_Daemon_a.txt"/>
+  </option>
+  <option id="temp">
+    <label>Temporary files</label>
+    <description>Delete the temporary files</description>
+    <action command="delete" search="file" path="$Temp\Miro-downloader.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log.1"/>
```

Comment:
Lines 60-62 deletes three log files that look like the three deleted on lines 52-54. Would you agree lines 60-62 should be deleted?

Reply:
IMHO, no! One it's the option "temp", one is the option "log".

Reply:
I think a good principle is the same action shouldn't be repeated in the same cleaner file (either within the same option or in a different option). First, if there are options labeled "log" and "temporary files," it implies chosing one doesn't do the other. If you think for Miro they are similar, you could eliminate the "logs" option and put all the actions under "temporary files."

Reply:
I prefer to remove "Temporary files", again. I will do this.